hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/InfoSubcommand.java

```diff
@@ -107,8 +109,11 @@ public void execute(ScmClient scmClient) throws IOException {
       if (verbose) {
         LOG.info("Pipeline Info: {}", container.getPipeline());
       } else {
-        LOG.info("Pipeline id: {}", container.getPipeline().getId().getId());
+        LOG.info("Pipeline id: {}",
```

Comment:
This also can be reverted as write pipeline Id already maintains the original pipeline